kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -388,3 +390,20 @@ fn collect_codegen_items<'tcx>(gcx: &GotocCtx<'tcx>) -> Vec<MonoItem<'tcx>> {
         }
     }
 }
+
+fn symbol_table_to_gotoc(tcx: &TyCtxt, file: &Path) {
+    let output_filename = file.with_extension("symtab.out");
+    let input_filename = file.with_extension("symtab.json");
+
+    let args =
+        vec![input_filename.into_os_string(), "--out".into(), output_filename.into_os_string()];
+    // TODO get symtab2gb path from self
+    let mut cmd = Command::new("symtab2gb");
+    cmd.args(args);
+    if cmd.status().is_err() {
+        let err_msg =
+            format!("Failed symtab2gb on file {}.", file.with_extension("symtab.json").display());
```

Comment:
Maybe a more user friendly message might be nice here. "Failed to generate goto model for {}" or something like that.

Reply:
done